### DISCUSSION TITLE
How to call edge functions without auth header?

### DISCUSSION THREAD

**Discussion by LiamMaclean216** [supabase/supabase#11093] [Questions] [answered]
I would like to use an edge function as an API endpoint. How can I let anyone make calls to the function without needing the 'Authorization: Bearer ANON_KEY' header?

**Comment by GaryAustin1** [comment, marked as answer]
I assume this flag is what you want,
> By default, Edge Functions require a valid JWT in the authorization header. This header is automatically set when invoking your function via a Supabase client library.
> 
> If you want to use Edge Functions to handle webhooks (e.g. [Stripe payment webhooks](https://github.com/supabase/supabase/tree/master/examples/edge-functions/supabase/functions/stripe-webhooks) etc.), you need to pass the --no-verify-jwt flag when deploying your function.
>
